src/cli.rs

```diff
@@ -87,6 +88,12 @@ fn parse_param(kv: &str) -> Result<Param, std::io::Error> {
     }
 }
 
+fn parse_rule(rule: &str) -> Result<Rule, std::io::Error> {
+    use std::io::{Error, ErrorKind};
```

Comment:
primarily out of consistency with `parse_param` which is defined above. As for why it's that way in `parse_param`, iirc it's to avoid polluting the whole module with the `io::` imports, since they are not useful elsewhere (we use another error type in the project, so this way `Error` is not ambiguous in the rest of the file.